[2021-11-01 02:05:42,343] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: my_dag.create_table_passengers manual__2021-11-01T02:05:41.348999+00:00 [queued]>
[2021-11-01 02:05:42,357] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: my_dag.create_table_passengers manual__2021-11-01T02:05:41.348999+00:00 [queued]>
[2021-11-01 02:05:42,357] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-11-01 02:05:42,357] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-11-01 02:05:42,357] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-11-01 02:05:42,367] {taskinstance.py:1262} INFO - Executing <Task(PostgresOperator): create_table_passengers> on 2021-11-01 02:05:41.348999+00:00
[2021-11-01 02:05:42,371] {standard_task_runner.py:52} INFO - Started process 1654 to run task
[2021-11-01 02:05:42,373] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'create_table_passengers', 'manual__2021-11-01T02:05:41.348999+00:00', '--job-id', '77', '--raw', '--subdir', 'DAGS_FOLDER/checkModel.py', '--cfg-path', '/tmp/tmpn4g7hk2h', '--error-file', '/tmp/tmpxh6o5jn3']
[2021-11-01 02:05:42,374] {standard_task_runner.py:77} INFO - Job 77: Subtask create_table_passengers
[2021-11-01 02:05:42,403] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.create_table_passengers manual__2021-11-01T02:05:41.348999+00:00 [running]> on host 93c37aa78767
[2021-11-01 02:05:42,445] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=create_table_passengers
AIRFLOW_CTX_EXECUTION_DATE=2021-11-01T02:05:41.348999+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-11-01T02:05:41.348999+00:00
[2021-11-01 02:05:42,451] {base.py:79} INFO - Using connection to: id: postgres_default. Host: 172.23.0.3, Port: 5432, Schema: ***, Login: ***, Password: ***, extra: {}
[2021-11-01 02:05:42,454] {dbapi.py:225} INFO - Running statement: CREATE TABLE if not exists passengers (passengerid int, name text, sex text, age float, sibsp int, parch int, ticket text, fare float, cabin text, embarked text);, parameters: None
[2021-11-01 02:05:42,455] {postgres.py:71} INFO - NOTICE:  relation "passengers" already exists, skipping

[2021-11-01 02:05:42,465] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=create_table_passengers, execution_date=20211101T020541, start_date=20211101T020542, end_date=20211101T020542
[2021-11-01 02:05:42,505] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-11-01 02:05:42,545] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
